specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/logic.json

```diff
@@ -6606,6 +6630,14 @@
           "readOnly": true,
           "description": "Gets the access endpoint."
         },
+        "endpointsConfiguration": {
+          "$ref": "#/definitions/FlowEndpointsConfiguration",
+          "description": "The endpoints configuration."
+        },
+        "accessControl": {
+          "$ref": "#/definitions/FlowAccessControlConfiguration",
+          "description": "The access control configuration."
+        },
         "sku": {
           "$ref": "#/definitions/Sku",
           "description": "The sku."
```

Comment:
shoudnt sku be readonly?

Reply:
As per discussion offline, Sku is supported property for existing legacy workflow with Sku name != ResourceSkuName.Consumption. So i will leave it as it is.